lib/DBDefs.pm.sample

```diff
@@ -210,7 +210,7 @@ sub WEB_SERVER                { "www.musicbrainz.example.com" }
 # sub DB_STAGING_SERVER_DESCRIPTION { '' }
 
 # Only change this if running a non-sanitized database on a dev server,
-# e.g. http://test.musicbrainz.org.
+# e.g. http://beta.musicbrainz.org.
```

Comment:
Ah, the old message is from when test.mb used to have peoples' real passwords, a long time ago. So it makes sense to update this, but it's now a bit misleading too because beta isn't a dev server.

I'd also change "dev server" to "staging server" above, and I think it'd be helpful to add a short list of what this actually does, now that we're overloading it a bit:
 1. Makes the "all passwords have been reset to 'mb'" banner visible.
 2. Disables the IP lookup admin tool.

Reply:
Added in https://github.com/metabrainz/musicbrainz-server/pull/1552/commits/d93249ffc34b99266e91bf2093e566eebe3409d1